docs/MIGRATION_GUIDE_08.md

```diff
@@ -188,6 +188,47 @@ const endpoint = await serverNode.add(
 );
 ```
 
+### New:BridgedNodeEndpoint <--> Legacy:ComposedDevice
+To create a Composed device in a bridge context a structure container is needed to encapsulate the endpoints of the composed device. In the new API this is done by the BridgedNodeEndpoint class.
+
+An Endpoint of this type is added to the Aggregator endpoint together with it's Bridged Device Basic Information data.
+
+```javascript
+const composed = new Endpoint(BridgedNodeEndpoint, {
+    id: "composed-device", // unique ID of the endpoint here
+    bridgedDeviceBasicInformation: {
+        ...
+    },
+});
+
+const composedWC = new Endpoint(
+    WindowCoveringDevice.with(WindowCoveringServer.with("Lift", "AbsolutePosition", "PositionAwareLift")),
+    {
+        id: "composed-wc", // Unique id
+    },
+);
+const composedLight = new Endpoint(DimmableLightDevice, {
+    id: "composed-light", // Unique id
+});
+
+const aggregator = new Endpoint(AggregatorEndpoint, { id: "aggregator" });
+
+await serverNode.add(aggregator);
+
+await aggregator.add(composed);
+
+await composed.add(composedWC);
+await composed.add(composedLight);
+```
+
+For hints on options to add endpoints into a structure see next paragraph.
+
+### How to add endpoints into a structure?
+The new API provides basically two ways to add endpoints into a structure:
+
+* "Lazy" way: When using `endpoint.parts.add()` or when you provide the parts in the configuration of the Endpoint when creating it. While this is convenient it has, as of current 0.8.x versions, the disadvantage that the Endpoint is not directly initialized, and so you can not directly add event handlers or such to the endpoint. This is only done when the node is online or an endpoint was added with full initialization (see next point). is planned to be changed in future versions.
```

Comment:
Events will change but the endpoint still won't be fully initialized until added to an initialized patent so state is available

Reply:
Ok  I add state, and event we do remove when we do it